src/NodaTime.Test/Calendars/BclCalendars.cs

```diff
@@ -43,6 +49,8 @@ private static Calendar GetCalendar(string name)
         public static Calendar Julian => GetCalendar("JulianCalendar");
         public static Calendar Hijri => GetCalendar("HijriCalendar");
 
+        public static IEnumerable<Calendar> MappedCalendars =>
```

Comment:
Probably at least some trivial documentation would be nice: it's not obvious to me what this is.